Sources/UIViewExtensions.swift

```diff
@@ -565,9 +565,34 @@ extension UIView {
         self.layer.mask = mask
     }
 
-    /// EZSwiftExtensions
+    /**
+     EZSwiftExtensions - Mask square/rectangle UIView with a circular/capsule cover
+     */
     public func roundView() {
-        self.layer.cornerRadius = min(self.frame.size.height, self.frame.size.width) / 2
+        self.setCornerRadius(radius: min(self.frame.size.height, self.frame.size.width) / 2)
+        self.layer.borderWidth = 0
+    }
+    
+    /**
+     EZSwiftExtensions - Mask square/rectangle UIView with a circular/capsule cover, with a border of desired color and width around it
+     
+     - Parameter withBorderColor: Color of circuler border
+     - Parameter withBorderWidth: Width of circuler border
+     */
+    public func roundView(withBorderColor color: UIColor, withBorderWidth width: CGFloat) {
+        self.setCornerRadius(radius: min(self.frame.size.height, self.frame.size.width) / 2)
+        self.layer.masksToBounds = true
```

Comment:
I was under the impression that the mask will not resize along with the view is masksToBounds is false. Apparently I'm wrong :( I'll remove it

This is the code snippet that i used to experiment
```
override func viewDidLoad() {
        let view = UIView(frame: CGRect(x: 0, y: 0, w: 250, h: 250))
        view.roundView()
        view.backgroundColor = .green
        view.layer.masksToBounds = false // toggle here
        
        self.view.addSubview(view)
        if #available(iOS 10.0, *) {
            timer = Timer.scheduledTimer(withTimeInterval: 0.5, repeats: true, block: { (_timer) in
                view.snp.updateConstraints({ (make) in
                    make.height.equalTo(ez.screenHeight)
                    make.width.equalTo(ez.screenWidth)
                })
                
                self.view.setNeedsLayout()
                
                UIView.animate(withDuration: 2, animations: {
                    self.view.layoutIfNeeded()
                }, completion: { (completed) in
                    if completed {
                        view.snp.updateConstraints({ (make) in
                            make.height.equalTo(100)
                            make.width.equalTo(100)
                        })
                        
                        self.view.setNeedsLayout()
                    }
                    
                    UIView.animate(withDuration: 2, animations: {
                        self.view.layoutIfNeeded()
                    })
                })
            })
        } else {
            // Fallback on earlier versions
        }
        
    }
```